src/jwplayer/utils/updateRVParam.ts

```diff
@@ -0,0 +1,29 @@
+export function updateRVParam(adTagUrl: string, rv: number) {
```

Comment:
I believe you could use [URL SearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) to simplify the code. Though not sure what the performance cost would be since you would be:
- transforming the URL string to a URL object
- using the URL object's searchParams API to read and write `rv` 
- converting back to string